lib/ts/controllers/s-tooltip.ts

```diff
@@ -82,7 +82,7 @@ namespace Stacks {
             if (!popover) {
                 popover = document.createElement("div");
                 popover.id = popoverId;
-                popover.className = "s-popover s-popover__tooltip";
+                popover.className = "s-popover s-popover__tooltip pe-none";
```

Comment:
I don't anticipate needing to interact with a tooltip's contents, so this looks good to me! Hovering on regular popovers is... not great at the moment, so when we revisit that we can investigate this at the same time.